__tests__/App-test.js

```diff
@@ -9,6 +9,14 @@ import App from '../src/App';
 // Note: test renderer must be required after react-native.
 import renderer from 'react-test-renderer';
 
+jest.useFakeTimers();
```

Comment:
why it's necessary?

Reply:
If we use real timers the following errors emerged
```
/Users/uarlouski/Workspace/Repositories/vividusTestApp/node_modules/scheduler/cjs/scheduler.development.js:47
        throw e;
        ^

TypeError: _reactNative.BackHandler.addEventListener is not a function
    at /Users/uarlouski/Workspace/Repositories/vividusTestApp/node_modules/@react-navigation/native/lib/commonjs/useBackButton.tsx:9:38
    at commitHookEffectListMount (/Users/uarlouski/Workspace/Repositories/vividusTestApp/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:10345:26)
    at commitPassiveHookEffects (/Users/uarlouski/Workspace/Repositories/vividusTestApp/node_modules/react-test-renderer/cjs/react-test-renderer.development.js:10383:11)
...
```

```
ReferenceError: You are trying to `import` a file after the Jest environment has been torn down.

      at Object.BackHandler (node_modules/react-native/index.js:191:12)
      at node_modules/@react-navigation/native/lib/commonjs/useBackButton.tsx:9:26
      at commitHookEffectListMount (node_modules/react-test-renderer/cjs/react-test-renderer.development.js:10345:26)
...
```